x-pack/plugins/xpack_main/public/components/telemetry/opt_in_details_component.js

```diff
@@ -94,13 +103,21 @@ export class OptInExampleFlyout extends Component {
         >
           <EuiFlyoutHeader>
             <EuiTitle>
-              <h2>Cluster statistics</h2>
+              <h2>
+                <FormattedMessage
+                  id="xpack.main.telemetry.callout.clusterStatisticsTitle"
+                  defaultMessage="Cluster statistics"
+                />
+              </h2>
             </EuiTitle>
             <EuiTextColor color="subdued">
               <EuiText>
-                This is an example of the basic cluster statistics that we&rsquo;ll collect.
-                It includes the number of indices, shards, and nodes.
-                It also includes high-level usage statistics, such as whether monitoring is turned on.
+                <FormattedMessage
+                  id="xpack.main.telemetry.callout.clusterStatisticsDescription"
+                  defaultMessage="This is an example of the basic cluster statistics that we&rsquo;ll collect.
```

Comment:
Maybe let's use `'` instead of `&rsquo;`
```suggestion
                  defaultMessage="This is an example of the basic cluster statistics that we'll collect.
```